hawc/apps/assessment/templates/assessment/assessmentvalue_form.html

```diff
@@ -3,7 +3,7 @@
 {% load crispy_forms_tags %}
 
 {% block content %}
-<div class="d-flex justify-content-end">
+<div class="float-right">
```

Comment:
this fixes the extra vertical padding by floating to the right instead of a flex container